### ISSUE TITLE
Coordinates, City and Region are broken since 1.3.0

### ISSUE THREAD

**Issue by nezed** [bluesmoon/node-geoip#141]
Hi! I've found that some data in lookup result are lost in `1.3.0` and `1.3.1`

For example, lookup for IP `8.8.8.8` in `geoip-lite@1.3.0` returns:
```json
{
  "range": [ 134742016, 134750207 ],
  "country": "US",
  "region": "NA",
  "eu": "0",
  "timezone": "",
  "city": "",
  "ll": [ 0, 0 ],
  "metro": 0
}
```

while looking same IP in `geoip-lite@1.2.2` returns:
```json
{
  "range": [ 134744064, 134744319 ],
  "country": "US",
  "region": "CA",
  "city": "Mountain View",
  "ll": [ 37.386, -122.0838 ],
  "metro": 807,
  "zip": 94035
}
```

Updating database locally on `geoip-lite@1.3.0` goes successfully, but result of lookup still missing some data, like:

- ll
- city
- region

I've tested a lot of IP's of greatly known locations and services, and i pretty sure that `1.3.0` and `1.3.1` versions contains bug

I've made a little demo to check result between `1.3` and `1.2`:
- https://runkit.com/embed/hz310no7aqmv

<img src="https://user-images.githubusercontent.com/1659008/42737357-a7d9b194-887a-11e8-9673-48a32afb9ea6.png" width=383 />

We're using `ll` coords in our app to resolve IATA cities and airports, so this feature is very important for us.
Hope you know how to find a problem, and thanks for your attention!

**Comment by nezed** [comment]
<img width=417 src="https://user-images.githubusercontent.com/1659008/42737440-077b7e4c-887c-11e8-9880-b53705aac160.png" />

**Comment by bluesmoon** [comment]
Yes, this is a side effect of #138.

**Comment by nezed** [comment]
Do you have any plans to fix coordinates in future?

**Comment by bluesmoon** [comment]
It's not up to us. Maxmind took it out.

**Comment by jasonrhodes** [comment]
But this was published as a _minor_ version bump and it totally changes the API of the library. Can you instead release this as a major version bump for geoip2, and as a prerelease candidate until they get all of the values they are going to have (like lat/lng)? If I understand what happened correctly, I think this severely broke a lot of code, especially publishing lat/lng as `[ 0, 0 ]`

**Comment by bluesmoon** [comment]
Yeah I suppose this should have been version 2.0.0.

**Comment by nezed** [comment]
@bluesmoon Please, correct me if i'm wrong.
I've checked maxmind's csv files, and as i see:

Matching location from IP in v1 format based on `locId`:
```sh
▶ head GeoLiteCity_20180327/GeoLiteCity-Blocks.csv
startIpNum,endIpNum,locId
"16777216","16777471","617943"
"16777472","16778239","104084"
"16778240","16779263","17"
```
And `locId` definitions has coords
```sh
▶ head GeoLiteCity_20180327/GeoLiteCity-Location.csv
locId,country,region,city,postalCode,latitude,longitude,metroCode,areaCode
1,"O1","","","",0.0000,0.0000,,
2,"AP","","","",35.0000,105.0000,,
3,"EU","","","",47.0000,8.0000,,
```

In v2 format i see new, and more identifiers for every subnet, even including coords:
```sh
▶ head GeoLite2-City-Blocks-IPv4.csv
network,geoname_id,registered_country_geoname_id,represented_country_geoname_id,is_anonymous_proxy,is_satellite_provider,postal_code,latitude,longitude,accuracy_radius
1.0.0.0/24,7302630,2077456,,0,0,3805,-38.0263,145.3069,1000
1.0.1.0/24,1810821,1814991,,0,0,,26.0614,119.3061,50
1.0.2.0/23,1810821,1814991,,0,0,,26.0614,119.3061,50
```
```sh
▶ head GeoLite2-City-CSV_20180703/GeoLite2-City-Blocks-IPv6.csv
network,geoname_id,registered_country_geoname_id,represented_country_geoname_id,is_anonymous_proxy,is_satellite_provider,postal_code,latitude,longitude,accuracy_radius
600:8801:9400:5a1:948b:ab15:dde3:61a3/128,5363990,,,0,0,91941,32.7596,-116.9940,100
2000:db8::/32,5332921,,,0,0,93614,37.2502,-119.7513,100
2001:200::/41,1864518,1861060,,0,0,182-0025,35.6556,139.5522,20
```
And also geolocations definition files in 8 (!!!) locales for `geoname_id` lookup:
```sh
▶ head GeoLite2-City-CSV_20180703/GeoLite2-City-Locations-en.csv
geoname_id,locale_code,continent_code,continent_name,country_iso_code,country_name,subdivision_1_iso_code,subdivision_1_name,subdivision_2_iso_code,subdivision_2_name,city_name,metro_code,time_zone,is_in_european_union
5819,en,EU,Europe,CY,Cyprus,02,Limassol,,,Souni,,Asia/Nicosia,1
49518,en,AF,Africa,RW,Rwanda,,,,,,,Africa/Kigali,0
49747,en,AF,Africa,SO,Somalia,BK,Bakool,,,Oddur,,Africa/Mogadishu,0
```

-----

As is see, getting coordinates for subnet becomes more easier in GeoLite2, and looking up for location description still available too

**Comment by bluesmoon** [comment]
yes, the GeoIP v1 files had lookup for IP.  I'll have to defer to @ar2rsawseen for the GeoIP v2 related issues.

**Comment by ar2rsawseen** [comment]
So checked lattitude and longitue are now working, added PR
https://github.com/ar2rsawseen/node-geoip/pull/142

But maybe before merging let's discuss other values.

So region, there is no field like that anymore, what would be the substitute for it?
Similarly city is taken from city_name field now.

Also what about other properties, maybe there are some useful too? Like I added area precision for coordinates as separate property.

**Comment by ar2rsawseen** [comment]
If there are no more suggestions, then I guess we can merge it

**Comment by bluesmoon** [comment]
@ar2rsawseen I'd left some review comments on the PR.

**Comment by ar2rsawseen** [comment]
Whoops, sorry I though I already fixed that, my bad, committing now

**Comment by mvala** [comment]
Is it possible to make release of 1.3.2 with this fix? Thanks
